lib/msal-common/src/unifiedCache/generator/TokenCacheGenerator.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AccessTokenEntity } from "../entities/AccessTokenEntity";
+import { IdTokenEntity } from "../entities/IdTokenEntity";
+import { RefreshTokenEntity } from "../entities/RefreshTokenEntity";
+import { AuthenticationResult } from "../../response/AuthenticationResult";
+
+export class TokenCacheGenerator {
```

Comment:
nit: TokenCacheEntityFactory

Reply:
This does not exist anymore as all token generation logic is pushed to the corresponding entity files